e2e/tests/apikey-lifecycle.spec.ts

```
@@ -349,3 +357,157 @@ test.describe('API Key Lifecycle', () => {
+      execSync(
+        `timeout 30 bash -c 'until kubectl get apikey ${keyName} -n ${consumerNs} -o jsonpath="{.status.conditions}" | grep -q Expired; do sleep 2; done'`,
+        { stdio: 'inherit' },
+      );
+
+      // Step 7: reload → Expired badge + "Expired (date)" in Expires column
+      await spaNavigate(page, `/kuadrant/apikeys/ns/${consumerNs}`);
+      await page.waitForLoadState('networkidle');
+      const expiredRow = page.locator(`tr:has-text("${keyName}")`);
+      await expect(
+        expiredRow.locator('td[data-label="status"]:has-text("Expired")'),
+      ).toBeVisible({ timeout: 15_000 });
```

ESLint

[error] Replace `⏎········expiredRow.locator('td[data-label="status"]:has-text("Expired")'),⏎······).toBeVisible({·timeout:·15_000` with `expiredRow.locator('td[data-label="status"]:has-text("Expired")')).toBeVisible({⏎········timeout:·15_000,⏎·····`

(prettier/prettier)